src/figuresClasses.ts

```diff
@@ -1,19 +1,90 @@
+type Color = 'red' | 'green' | 'blue';
+
+enum Shape {
+  Triangle = 'triangle',
+  Circle = 'circle',
+  Rectangle = 'rectangle',
+}
+
 export interface Figure {
+  color: Color,
+  shape: Shape,
 
+  getArea(): number,
 }
 
-export class Triangle {
+function isTheBigestSide(a: number, b: number, c: number): boolean {
```

Comment:
Bad name for function, name should start with a verb and should describe what this function do